librariesio-url-parser.gemspec

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "librariesio-url-parser"
+
+Gem::Specification.new do |spec|
+  spec.name          = "librariesio-url-parser"
+  spec.version       = LibrariesioURLParser::VERSION
+  spec.authors       = ["Andrew Nesbitt"]
+  spec.email         = ["andrewnez@gmail.com"]
```

Comment:
I'm not sure if there have ever been any actual rules, but it seems like you could add yourself since you extracted it to a gem 🤷🏻